index.js

```diff
@@ -199,7 +199,7 @@ module.exports = function (app) {
   }
 
   function setKey (info, dest_key, source_key) {
-    var val = _.get(app.signalk.self, source_key)
+    var val = app.getSelfPath(source_key)
     if (val !== undefined) info[dest_key] = val
```

Comment:
Setting values like this won’t work in the future either. Do we need  `setSelfPath`?

A bit different matter, so does not prevent merging this.

Reply:
I don't think so. I've never seen anything that sets values that way. Should be done with delta's, right?

Reply:
Actually, might be a nice shorthanded way for plugins to send deltas.

Reply:
Deltas would be the way, yes.